lib/formatDate.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const twoPaddedNumbers = new Array(60).fill(0).map((v, i) => i.toString().padStart(2, '0'))
+const threePaddedNumbers = new Array(1000).fill(0).map((v, i) => i.toString().padStart(3, '0'))
+
+const monthLookup = new Array(12).fill(0).map((v, i) => (i + 1).toString().padStart(2, '0'))
+const tzOffsetLookup = {}
+
+for (let i = -900; i < 900; i += 15) {
+  tzOffsetLookup[i] = (i > 0 ? '-' : '+') + ('' + (Math.floor(Math.abs(i) / 60) * 100 + (Math.abs(i) % 60))).padStart(4, '0')
```

Comment:
What does the `'' +` do?

Reply:
Coerces the value to a string.

Reply:
Yup. Coercing the number to a string ;)

Reply:
I guess I can savely resolve this conversation ;)